app/models/image.rb

```diff
@@ -5,9 +5,25 @@ class Image < BaseModel
   self.abstract_class = true
 
   include PublicAsset
+  include Fixerable
+
+  UPLOADED = 'uploaded'.freeze
+  NOTFOUND = 'not found'.freeze
+  INVALID  = 'invalid'.freeze
+  FAILED   = 'failed'.freeze
+  COMPLETE = 'complete'.freeze
+
+  alias_attribute :upload, :upload_path
 
   mount_uploader :file, ImageUploader, mount_on: :filename
-  alias_attribute :upload, :remote_file_url
+  skip_callback :commit, :after, :remove_file! # don't remove s3 file
```

Comment:
same comment as above - and since we generally always want to keep files, I think maybe a global config makes more sense.